WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -141,7 +141,7 @@ struct DestructiveButtonRow: ImmuTableRow {
         cell.textLabel?.text = title
         cell.textLabel?.numberOfLines = 0
         cell.textLabel?.lineBreakMode = .byWordWrapping
-
+        cell.selectionStyle = .none
```

Comment:
By setting this the row shows nothing when tapping it - was this committed by mistake?